.github/workflows/build.yml

```diff
@@ -456,6 +456,116 @@ jobs:
           ../scripts/xcodebuild.sh ${{ steps.configure.outputs.project-directory }}/TemplateExample.xcworkspace build
         working-directory: template-example
     timeout-minutes: 60
+  visionos:
+    name: "visionOS"
+    runs-on: macos-14
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          filter: blob:none
+          fetch-depth: 0
+      - name: Set up toolchain
+        uses: ./.github/actions/setup-toolchain
+        with:
+          platform: visionos
+          project-root: example
+          cache-key-prefix: example
+          xcode-developer-dir: /Applications/Xcode_15.2.app
```

Comment:
This was added because the default of Github Actions right now (Xcode 15.0) doesn't contain the visionOS SDK